### ISSUE TITLE
Error when setting datasource of combo editor provider of igGrid

### ISSUE THREAD

**Issue by igMPeterson** [IgniteUI/igniteui-angularjs#108]
When setting the datasource of the combo editor of the igGrid it seems to think it is a url and throws and error. When you setup and igCombo outside of the grid the same way it works. To get the error edit the "LastName" column.

[igGrid_comboEditorDataSource.zip](https://github.com/IgniteUI/igniteui-angular/files/1085513/igGrid_comboEditorDataSource.zip)

**Comment by dkamburov** [comment]
@zdrawku, @igMPeterson I'm unable to download the sample, can you reattach it?

**Comment by zdrawku** [comment]
I am unable to download it too, @igMPeterson if this issue is reported from a client, could you check the case for a sample, or does this injury still persist.

**Comment by mpavlinov** [comment]
@igMPeterson The attached sample cannot be downloaded. Can you re-attach the sample, please.

**Comment by dkamburov** [comment]
I found the original sample. Here's the [link](https://github.com/IgniteUI/igniteui-angularjs/files/1085513/igGrid_comboEditorDataSource.zip) to it.

@igMPeterson You were right. In the following case the datasource of the igCombo is considered as a string.
The issue in this is sample is related to the fact that, all of the nested configuration is used only for configuration purpose. They are not used as bindings, only top level options can be used as bindings.

So basically this - 
```
<editor-options data-source="employees" ...>
</editor-options>
```
Should be converted to, so that the datasource can be evaluated:
```
<editor-options data-source="{{employees}}" ...>
</editor-options>
```

I'm also attaching the modified sample.
[igGrid_comboEditorDataSource.zip](https://github.com/IgniteUI/igniteui-angularjs/files/2876282/igGrid_comboEditorDataSource.zip)

**Comment by dkamburov** [comment]
Closing this as it is by design, if there are questions on it, please re-open it.
